src/utils.h

```diff
@@ -0,0 +1,6 @@
+/* Get system time*/
+#pragma once
+#include <sys/time.h>
+#include <time.h>
```

Comment:
`struct timeval` is defined in header `<sys/time.h>>`.
See https://pubs.opengroup.org/onlinepubs/7908799/xsh/systime.h.html